packages/components/src/internal/components/samples/actions.ts

```diff
@@ -160,8 +165,8 @@ export function fetchSamples(
                 [id, sampleColumn.fieldKey],
                 List([
                     {
-                        displayValue: rows.getIn([id, sampleColumn.lookup.displayColumn, 'value']),
-                        value: rows.getIn([id, sampleColumn.lookup.keyColumn, 'value']),
+                        displayValue: rows.getIn([id, displayValueKey, 'value']),
```

Comment:
Can remove our usage of `fromJS` here and prevent against case-sensitivity mapping errors if we use `caseInsensitive` against the plain old row object:

```ts
const rows = models[key];
// later...
displayValue: caseInsensitive(rows[id], displayValueKey)?.value,
value: caseInsensitive(rows[id], valueKey)?.value,
```